pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions and manipulate strings.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical or math expression with an operator('+','-','*','&','|' and etc.), special symbols should be escaped:
```

Comment:
ok, will correct this. 
dunnow how to get rid of this overall cumbersome construction, it is seems  that I have to leave it like it is with your corrections.